src/components/DocumentationTopic.vue

```diff
@@ -42,10 +42,10 @@
         >
           <component :is="titleBreakComponent">{{ title }}</component>
           <template #after v-if="isSymbolDeprecated || isSymbolBeta">
-          <small
-          :class="tagName"
-          :data-tag-name="tagName"
-          />
+            <small
```

Comment:
Not important but maybe you meant to only indent the prop assignments here?

Reply:
I meant to indent the whole thing because it is within the `template`. I'm not sure if we just forgot to indent it originally or it's meant to be on the same level as the `template`.